packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/authn/requests/FlowStartRequest.java

```diff
@@ -38,7 +36,7 @@ public static class Builder {
 
 		private String cbURI;
 		private String email;
-		private FlowType flowType;
+		private FlowType[] flowType;
 		private IDProvider provider;
```

Comment:
The v2 endpoint no longer takes an IDProvider. Missing an invitation field here now:
![Screenshot 2023-10-19 at 5 44 26 PM](https://github.com/pangeacyber/pangea-java/assets/10459856/7fadafd5-8ec3-4ea6-96b1-0df974683669)

Reply:
fixed! Thanks!